docker-compose.yml

```diff
@@ -12,6 +12,18 @@ services:
     ports:
       - 5000:5000
 
+  tweet-api:
+    build:
+      args:
+        FLASK_DEBUG: ${FLASK_DEBUG:-0}
+      context: tweet-api
+    environment:
+      FLASK_DEBUG: ${FLASK_DEBUG:-0}
+    volumes:
+      - ./tweet-api/:/app
+    ports:
+      - 5000:5000
```

Comment:
However, here I'd suggest `- ${PORT-5000}:5000` to make it easier to stage multiple versions side by side

Reply:
Or maybe `- ${API_PORT-5000}:5000` since we also have a frontend port below 👍🏻